meet-data/nzpf/1621/entries.csv

```diff
@@ -0,0 +1,15 @@
+Name,Equipment,Sex,Division,BodyweightKg,WeightClassKg,Age,Squat1Kg,Squat2Kg,Squat3Kg,BestSquatKg,Bench1Kg,Bench2Kg,Bench3Kg,BestBenchKg,Deadlift1Kg,Deadlift2Kg,Deadlift3Kg,BestDeadliftKg,TotalKg,Event,Place
+Orla Harris,Raw,F,Juniors,56.5,57,19,107.5,115,120,120,67.5,-75,-75,67.5,105,115,-120,115,302.5,SBD,1
```

Comment:
How was Raw vs Equipped determined here? In the results file, everyone is marked equipped.

Reply:
U-EQP is Raw and EQP is single-ply in the results file.